versioned_sidebars/version-8.1-sidebars.json

```diff
@@ -1095,12 +1095,7 @@
             },
             {
               "type": "link",
-              "label": "Update notes (3.9. to 3.10)",
-              "href": "/optimize/self-managed/optimize-deployment/migration-update/3.9-to-3.10/"
-            },
```

Comment:
I actually don't know how this wasn't causing link-checking errors. This URL doesn't exist, in staging or prod.

Reply:
Strange, it's in the sidebar for both 3.9 and `next`, but in 3.9 it doesn't find a page at `https://stage.docs.camunda.io/optimize/self-managed/optimize-deployment/migration-update/3.9-to-3.10/`.

The page does exist at `https://stage.docs.camunda.io/optimize/next/self-managed/optimize-deployment/migration-update/3.9-to-3.10/`. 

I'm also surprised it was missed, but let's keep this in mind today as we work toward the 8.2 release. IMO it shouldn't be in 8.1 at all (or 3.9 in Optimize version speak).